sdk/js/src/token_bridge/attest.ts

```diff
@@ -197,3 +203,79 @@ export async function attestFromAlgorand(
 
   return txs;
 }
+
+/**
+ * Attest an already created asset
+ * If you create a new asset on near and want to transfer it elsewhere,
+ * you create an attestation for it on near... pass that vaa to the target chain..
+ * submit it.. then you can transfer from near to that target chain
+ * @param client An Near account client
+ * @param tokenBridge The account for the token bridge
+ * @param asset The account for the asset
+ * @returns sequenceNumber
```

Comment:
Looks like it returns more than a sequenceNumber?

Reply:
also returns the emitter